src/Bridge/Symfony/Bundle/Resources/config/doctrine_orm_http_cache_purger.xml

```diff
@@ -12,7 +12,10 @@
             <argument type="service" id="api_platform.http_cache.purger" />
             <argument type="service" id="api_platform.iri_converter" />
             <argument type="service" id="api_platform.resource_class_resolver" />
-
+            <argument>null</argument>
```

Comment:
Why the `api_platform.property_accessor` service is not used?

Reply:
I'm not sure we should add it here indeed